docs/using-radicle/pushing-changes.md

```diff
@@ -0,0 +1,42 @@
+---
+id: pushing-changes
+title: Pushing changes
+---
+
+To push code to Radicle, you first need to make sure your shell is configured
+correctly by following the instructions in the [Getting Started][gs] section to
+add the Radicle remote helper.
+
+After the Radicle remote helper is configured, pushing to Radicle is as simple
+as pushing to any other remote, with the Radicle network being known as 'rad'.
+After changes are added and commited in git, they can be pushed to the Radicle
+remote via `git push rad`. This will push your upstream tracking branch. 
+
+Radicle projects also support [branches][br]. Specific branches can be pushed to
+Radicle with:
+
+`git push rad <branch name>`
+
+[DO WE NEED TO ADD SOMETHING ABOUT -U HERE]
```

Comment:
We should expand that if the user wants tracking for their branch to be set up for the radicle remote they should invoke:

```
git push -u rad <branch_name>
```

As expected with any other git code host.

Reply:
```suggestion
To set up branch tracking, run:

`git push -u rad <branch_name>` 

as you would with any other Git code hosting platform.
```